.vscode/settings.json

```diff
@@ -9,4 +9,10 @@
     "[python]": {
         "editor.defaultFormatter": "charliermarsh.ruff",
     },
+    "python.linting.mypyEnabled": true,
```

Comment:
This needs to be conditional, which will require making `settings.json` a jinja expanded template. Please make a separate issue for just this line and we will discuss there

Reply:
but is it worth it? I'm not opinionated, we could close this